### PR TITLE
Enable filebeat

### PR DESCRIPTION
**Pull request by afda16**

### CODE REVIEW

File: hieradata/integration.yaml [hunk 1]
```diff
@@ -31,6 +31,9 @@ govuk::apps::whitehall::basic_auth_credentials: "%{hiera('http_username')}:%{hie
 govuk::apps::whitehall::highlight_words_to_avoid: true
 govuk::apps::whitehall::enable_tagging_to_new_taxonomy: "yes"
 
+govuk_beat::hosts:
```
Comment (afda16) [inline hieradata/integration.yaml]: I'm not sure why we shouldn't just override govuk_beat::hosts in the credentials repo, does the logstash_host variable already exist?

File: hieradata/integration.yaml [hunk 2]
```diff
@@ -31,6 +31,9 @@ govuk::apps::whitehall::basic_auth_credentials: "%{hiera('http_username')}:%{hie
...
+  - "%{hiera('logstash_host')}:5566"
```
Comment (afda16) [inline hieradata/integration.yaml]: I'm not sure why we shouldn't just override govuk_beat::hosts in the credentials repo, does the logstash_host variable already exist?

Comment (SamLR) [inline hieradata/integration.yaml]: @afda16 overriding it in the credentials repo is fine. This was me being bad at puppet

**Comment by surminus** [comment]
Might be good to add `/var/log/syslog`, too?

**Comment by afda16** [comment]
Added syslog

**Pull request merged**